mule-maven-plugin/src/main/java/org/mule/tools/maven/utils/Exclusion.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.tools.maven.utils;
+
+public class Exclusion extends org.apache.maven.model.Exclusion implements org.mule.tools.api.packager.packaging.Exclusion {
```

Comment:
Is not a very good practice to name an implementation after its interface (though we do it some times ) 
Given the both are new and ours cane we rename this to DefaultArtifact or something like that?